game-core/src/main/java/games/strategy/net/IpFinder.java

```diff
@@ -79,26 +79,5 @@ public static InetAddress findInetAddress() throws SocketException, UnknownHostE
     }
     // all else fails, return localhost
     return InetAddress.getLocalHost();
-  } // end static findInetAddress()
-
-  private static boolean isPublicNetworkAddress(final InetAddress address) {
-    // stupid java signed byte type
-    final byte octet254 = (byte) 0xFE;
-    final byte[] bytes = address.getAddress();
-    // ip 4
-    if (bytes.length == 4) {
-      // http://en.wikipedia.org/wiki/Private_network
-      final byte octet169 = (byte) 0xA9;
-      final byte octet168 = (byte) 0xA8;
-      final byte octet172 = (byte) 0xAC;
-      final byte octet192 = (byte) 0xC0;
-      return (bytes[0] != 10) && (bytes[0] != octet172 || bytes[1] < 16 || bytes[1] > 31)
-          && (bytes[0] != octet192 || bytes[1] != octet168) && (bytes[0] != octet169 || bytes[1] != octet254);
```

Comment:
The old implementation also checked for the subnet 169.254.0.0/16.  This subnet is not covered by `Inet4Address#isSiteLocalAddress()`, but it is included in `Inet4Address#isLinkLocalAddress()`.  Do we need to preserve that behavior?  If so, I would suggest keeping this method and simply changing the implementation to delegate to those two methods.